src/test/scala/dynamodb/benchmark/BenchmarkSpec.scala

```diff
@@ -128,37 +129,40 @@ class BenchmarkSpec extends AnyWordSpec with Matchers with BeforeAndAfterAll {
     } / xs.size)
   }
 
-
-  "The cluster" should {
-    /**
-     * The cluster is kept online during all the tests, so make sure that you use unique keys for each test.
-     */
-    "survive get benchmark" in {
-      val coordinator = getCoordinatorUrlForKey("myKey")
-      val coordinatorUrl = hostToUrl(coordinator)
-      post(coordinatorUrl, Value("myKey", "myValue"))
-        .body should be("Value added")
-      val res = for (_ <- 0 until 10000) yield getBench("myKey")
+  it should "survive get benchmark" in {
+    val coordinator = getCoordinatorUrlForKey("myKey")
+    val coordinatorUrl = hostToUrl(coordinator)
+    post(coordinatorUrl, Value("myKey", "myValue"))
+      .body should be("Value added")
+    val start = System.nanoTime()
+    val resFutures = for (_ <- 0 until 10000) yield getBench("myKey")
```

Comment:
Werkt dit? Volgens mij moet je alsnog expliciet wachten totdat je klaar bent met al je acties voordat je `end` gaat vullen.

Reply:
is gefixed nu